app/views/organizations/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<p>
+  Hello, <%= @org.name %>
+</p>
+
+<p>
+<% if @organization_campaign %>
+  Your current campaign is <%= @organization_campaign.campaign.name %>
+
+  <table>
+    <tr>
```

Comment:
Also, I typically do a `<thead>` here. Since this table won't get very big (old campaigns fall off) probably not necessary but I use thead for consistency